conda-forge.yml

```
@@ -4,3 +4,5 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+build_platform: {osx_arm64: osx_64, linux_aarch64: linux_64}
+test_on_native_only: true
```

`test_on_native_only` is deprecated and the following is the (new) equivalent
```suggestion
test: native_and_emulated
```